### PR TITLE
Issue #1297 - Intranet NuGet Gallery with Update from Official Source

### PR DESCRIPTION
**Pull request by greggbjensen**
I have added the ability to set a flag that enables the NuGet Gallery as an intranet site. This sets both branding, and the ability to update your intranet NuGet Gallery with packages directly from the official source. I think this is a feature that a lot of companies are looking for. Managed access to everything on the official source, with the ability to host internal NuGet packages.

I added pages to browse the official NuGet source to check which packages you have and download new ones.  This will allow companies to easily keep them in sync.

### CODE REVIEW

**Comment by phatcher** [comment]
@greggbjensen : I finally got around to cleaning up the merge from the current master implementation, it's on phatcher-1297-intranet and I've submitted a pull request

File: Website/Areas/Admin/Controllers/ImportController.cs [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+using NuGetGallery.NuGetService;
+
+namespace NuGetGallery.Areas.Admin.Controllers
+{
+    using V2FeedPackage = NuGetGallery.NuGetService.V2FeedPackage;
+
+    /// <summary>
+    /// Controller for importing directly from the NuGet official package source.
+    /// </summary>
+    public partial class ImportController : AdminControllerBase
```
Comment (xavierdecoster) [inline Website/Areas/Admin/Controllers/ImportController.cs]: In light of the upcoming v3, we can't take this change. Please consider a proxy solution on top of v3 if you need this functionality.

**Comment by xavierdecoster** [comment]
This PR contains several things:
* improvements to support intranet branding
* functionality to import packages from another (v2) feed

We envision improved branding capabilities as well as support for the v3 protocol. We can't take this in as-is, but would you consider a separate PR that tackles the above functionality separately?

Thanks!

**Comment by xavierdecoster** [comment]
Closing this PR and moving the discussion to the newer (updated) PR here: https://github.com/NuGet/NuGetGallery/pull/2382

**Pull request closed without merging**